warehouse/query-core/src/test/java/datawave/query/RebuildingScannerTestHelper.java

```diff
@@ -5,6 +5,9 @@
 import datawave.accumulo.inmemory.InMemoryInstance;
 import datawave.accumulo.inmemory.InMemoryScanner;
 import datawave.accumulo.inmemory.ScannerRebuilder;
+import datawave.query.attributes.Document;
+import datawave.query.function.deserializer.KryoDocumentDeserializer;
+import datawave.accumulo.inmemory.*;
```

Comment:
No star imports, same comment for several other files.

Reply:
Do we not have a rule set up to enforce this in the pom's? If not, we should.